packages/client/src/client/App/Footer/ContactUsButton/ContactUs.tsx

```diff
@@ -28,10 +29,10 @@ export const ContactUs = () => (
       onClick={() => {
         window.open(`mailto:${EMAIL}`)
       }}
+      paddingBottom="xs"
     >
-      <Typography>{EMAIL}</Typography>
+      {EMAIL}
```

Comment:
minor point - what is the point of `Link` here .. I'd rather just have the plain Typography element inline - like this, it makes me think that Link has some "special powers" 🙂